app/views/listings/form/_currency_selector.haml

```diff
@@ -1,10 +1,3 @@
 .currency-selector
-  - if @current_community.available_currencies
-    - currencies = @current_community.available_currencies.gsub(" ","").split(",")
-    - if currencies.size > 1
-      = form.select :currency, currencies, { :selected => (@listing.currency || currencies.first) }
-    - else
-      = Money.new(1, currencies[0]).symbol
-      = form.hidden_field :currency, :value => currencies[0]
-  - else
-    = form.select :currency, major_currencies(Money::Currency.table), { :selected => (@listing.currency || "EUR") }
+  = Money.new(1, @current_community.currency).symbol
+  = form.hidden_field :currency, :value => @current_community.currency
```

Comment:
Now that a marketplace has only one currency, we don't need to circulate that information through view layer. This hidden field could be removed, and the `listings_controller` could be changed so that it would read `@current_community.currency` instead of `params[:currency]`.